### PR TITLE
Single video page

### PR DESCRIPTION
**Pull request by Ankur9669**
Files to check:
1. SinglePage.js
2. LeftContainer.js
3. RightContainer.js
4. leftcontainer.css
5. rightcontainer.css

image

### CODE REVIEW

File: FitVid/src/Pages/SingleVideoPage/LeftContainer/leftcontainer.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.left-container{
+   grid-column: 1 / 2;
+   /* min-height: 120vh; */
```
Comment (Naman-Saxena1) [inline FitVid/src/Pages/SingleVideoPage/LeftContainer/leftcontainer.css]: If not using, can remove commented

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Code looks great 👍 
I do have 2 nit picks for UI design
### 1. Alignment should be same for consistent design, both should start from same height
### 2. On hover of options, it should be cursor: pointer to indicate it is clickable
InkedCapture3_LI

File: FitVid/src/Pages/SingleVideoPage/LeftContainer/leftcontainer.css (hunk 1 above)
Comment (Ankur9669) [reply FitVid/src/Pages/SingleVideoPage/LeftContainer/leftcontainer.css]: Done

**Review by Priya-Patel31 — APPROVED** [review]
The code looks good . Could'nt find anything to suggest . You can go ahead merge .

**Pull request merged**